src/core/settings.hpp

```diff
@@ -1041,8 +1041,9 @@ namespace big
 			int sort_direction = 0;
 
 			bool replace_game_matchmaking = false;
+			bool without_host_modders     = false;
 
-			NLOHMANN_DEFINE_TYPE_INTRUSIVE(session_browser, region_filter_enabled, region_filter, language_filter_enabled, language_filter, player_count_filter_enabled, player_count_filter_minimum, player_count_filter_maximum, sort_method, sort_direction, replace_game_matchmaking, pool_filter_enabled, pool_filter)
+			NLOHMANN_DEFINE_TYPE_INTRUSIVE(session_browser, region_filter_enabled, region_filter, language_filter_enabled, language_filter, player_count_filter_enabled, player_count_filter_minimum, player_count_filter_maximum, sort_method, sort_direction, replace_game_matchmaking, pool_filter_enabled, pool_filter, without_host_modders)
```

Comment:
`without_host_modders`, this variable just doesn't sound right. I obviously know what it means within the context of this PR.

My suggestion would be to change this to `exclude_modder_sessions`.